CHANGELOG.md

```diff
@@ -25,6 +25,15 @@ Classify the change according to the following categories:
     ### Deprecated
     ### Removed
 
+## new-dc-couple
+### Added
+- Add option to model DC-coupled **PV** and **ElectricStorage**
+    - Add **PV** input **dc_coupled_with_storage** and **ElectricStorage** input **dc_coupled**, which both default to _false_
```

Comment:
@hdunham I haven't looked through the code yet, so this maybe become more clear to me, but I'm a bit confused initially by these inputs. 

Why not just have one input for dc_coupled pv and storage rather that two? It seems like having two could allow these inputs to conflict (if one is true and the other is false, what happens?). The changes should only ever have an impact if you model both PV and BESS together; if only one or the other is included in the model, the results should just use the typical inverter setup, right?

Reply:
Is your thought that with multiple storages you might want to be able to choose some with which PV is DC coupled while some have their own separate inverters? Let's discuss on our call

Reply:
Yes exactly